src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import React, { useState, FormEvent } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { AddedTodos } from './AddedTodos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+export const App: React.FC = () => {
+  const [titleInput, setTitleInput] = useState('');
+  const [userSelect, setUserSelect] = useState(0);
+  const [titleFull, setTitleFull] = useState(true);
+  const [userChosen, setUserChosen] = useState(true);
+  const [todoId, setTodoId] = useState(
+    Math.max(...todosFromServer.map(todo => todo.id)),
+  );
+
+  const toDosUser = todosFromServer.map((todo) => {
+    return {
+      ...todo,
+      user: usersFromServer
+        .find((userFromId): boolean => userFromId.id === todo.userId),
+    };
+  });
+
+  const submitForm = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setTitleFull(!!titleInput);
+    setUserChosen(!!userSelect);
+
+    if (titleInput && userSelect) {
+      setTodoId(todoId + 1);
```

Comment:
If the new state is computed using the previous state, you can pass a function to setTodoId